src/main/java/org/lsc/beans/BeanComparator.java

```diff
@@ -320,10 +321,32 @@ private static LscModifications getUpdatedObject(
 
 				case REPLACE_VALUES:
 					if (attrStatus == PolicyType.FORCE) {
-						if (isModified(dstBean, dstAttrValues, toSetAttrValues)) {
+						// check if there are any extra values to be added
+						Set<Object> missingValues = SetUtils.findMissingNeedles(dstAttrValues, toSetAttrValues);
+						// check if there are any extra values to be removed
+						Set<Object> extraValues = SetUtils.findMissingNeedles(toSetAttrValues, dstAttrValues);
+
+						if((missingValues.size() + extraValues.size()) >= toSetAttrValues.size()) {
```

Comment:
I am not sure at which point one replace is more efficient than 1 add + 1 delete.

Maybe it can be a configuration parameter? The condition would be:

```
(missingValues.size() + extraValues.size()) >= max_value_modifications
```

It would make the LSC modifications more predictable.

However I understand that depending on the entry size has also its benefits. For example, with the previous condition, if we had an entry with 1000 values, and max_value_modifications = 100, then 50 adds and 51 adds would lead to a big replace, which may be less efficient than adding 50 values + removing 51 values.

Without further efficiency analysis, I think we can keep the initial proposition in this PR.